src/main/frontend/components/FilmShow.js

```diff
@@ -91,6 +91,7 @@ const FilmShow = props => {
       <ReviewTile
         key={review.id}
         review={review}
+        somethingElse={fetchFilm}
```

Comment:
This is a very interesting setup, having an async call being passed into the component.  I'd be a little worried about the call not finishing before the ReviewTile renders, (so you'd get an undefined variable), but if it passed your testing, let's try it out!

Reply:
On second thought, I think this can potentially get messy with the component lifecycle.  You'd be wasting resources on another fetch call - instead, you could try handing down a method like `removeReview` and in `FilmShow`, `removeReview(id)` would take an id, go into an existing state of `film`, and filter `film.reviews` to remove the review with the matching id.

Then, in `ReviewTile`, instead of calling `props.SomethingElse()`, you can get the deleted id as part of the fetch response body (using `await reponse.json()`) and call `props.removeReview(id)` to tell the parent to remove the review from state